test/conformance/helpers/broker_control_plane_test_helper.go

```diff
@@ -136,7 +136,17 @@ func triggerV1Beta1ReadyAfterBrokerIncludesSubURI(t *testing.T, triggerName, bro
 
 	client.WaitForResourceReadyOrFail(triggerName, testlib.TriggerTypeMeta)
 
-	tr, err := client.Eventing.EventingV1beta1().Triggers(client.Namespace).Get(context.Background(), triggerName, metav1.GetOptions{})
+	var tr *eventingv1beta1.Trigger
+	triggers := client.Eventing.EventingV1beta1().Triggers(client.Namespace)
+	err = client.RetryWebhookErrors(func(attempts int) (err error) {
+		var e error
+		client.T.Logf("Getting v1beta1 trigger %s", triggerName)
+		tr, e = triggers.Get(context.Background(), triggerName, metav1.GetOptions{})
+		if e != nil {
+			client.T.Logf("Failed to get trigger %q: %v", triggerName, e)
+		}
+		return err
+	})
 	if err != nil {
```

Comment:
After retrying we sometimes got an already exists error, can you add a check for that?

https://github.com/knative/eventing/blob/c5fabfd86e82c5b3a71de2c8fa2881f7067e8380/test/lib/creation.go#L81-L92

```suggestion
	if err != nil && !apierrors.IsAlreadyExists(err) {
```

The same applies for the others below.

Reply:
I only added retries for GET operations and I don't think we would get that error in this case.

And, all the other operations already have retries.

Let me know what you think @pierDipi

Reply:
It sounds good, thanks! 
/lgtm